cmd/system-probe/subcommands/run/command.go

```diff
@@ -250,8 +252,7 @@ func StopSystemProbeWithDefaults() {
 // startSystemProbe Initializes the system-probe process
 func startSystemProbe(log log.Component, statsd compstatsd.Component, telemetry telemetry.Component, sysprobeconfig sysprobeconfig.Component, rcclient rcclient.Component, wmeta optional.Option[workloadmeta.Component]) error {
 	var err error
-	var ctx context.Context
-	ctx, common.MainCtxCancel = context.WithCancel(context.Background())
+	_, common.MainCtxCancel = context.WithCancel(context.Background())
```

Comment:
I don't think anything is using `common.MainCtxCancel` now, so you can probably remove it.